doc/examples/reconst_msdki.py

```diff
@@ -0,0 +1,351 @@
+assuming that the degree the non-Gaussian diffusion decreases with tissue
+degeneration, this can be sensitive to general microstructural alterations.
+Although this cannot be use to distinguish different mechanisms of
+microstructural changes (e.g. axonal loss vs demyelination), the degree of
+non-Gaussian diffusion can provide insights on the general condition of tissue
+microstructure and provide useful markers to understanding, for instance, the
+relationship between brain microstructure changes and alterations in behaviour
+(e.g. [Price2017]_).
+
+Diffusion Kurtosis Imaging is one of the conventional ways to estimate the
+degree of non-Gaussian diffusion (see :ref:`example_reconst_dki`). However,
+as previously pointed [NetoHe2015]_, standard kurtosis measures do not only
+depend on microstructural properties but also on mesoscopic properties such as
+fiber dispersion or the intersection angle of crossing fibers.
+
+In the following example, we show how one can process the diffusion kurtosis
+from mean signals (also known as powder-averaged signals) and obtain a
+characterization of non-Gaussian diffusion independently to the degree of fiber
+organization [NetoHe2018]_. In the first part of this example, the properties
+of the measures obtained from the mean signal diffusion kurtosis imaging
+[NetoHe2018]_ are illustrated using synthetic data. Secondly, the mean signal
+diffusion kurtosis imaging will be applied to in-vivo MRI data.
+
+Let's import all relevant modules:
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+# Reconstruction modules
+import dipy.reconst.dki as dki
+import dipy.reconst.msdki as msdki
+
+# For simulations
+from dipy.sims.voxel import multi_tensor
+from dipy.core.gradients import gradient_table
+from dipy.core.sphere import disperse_charges, HemiSphere
+
+# For in-vivo data
+from dipy.data import fetch_cfin_multib
+from dipy.data import read_cfin_dwi
+from dipy.segment.mask import median_otsu
+
+"""
+===============================================================================
+Testing MSDKI in synthetic data
+===============================================================================
+
+We simulate representative diffusion-weighted signals using MultiTensor
+simulations (for more information on this type of simulations see
+:ref:`example_simulate_multi_tensor`). For this example, simulations are
+produced based on the sum of four diffusion tensors to represent the intra-
+and extra-cellular spaces of two fiber populations. The parameters of theses
+tensors are adjusted according to [NetoHe2015]_ (see also
+:ref:`example_simulate_dki`).
+"""
+
+mevals = np.array([[0.00099, 0, 0],
+                   [0.00226, 0.00087, 0.00087],
+                   [0.00099, 0, 0],
+                   [0.00226, 0.00087, 0.00087]])
+
+"""
+For the acquisition parameters of the synthetic data, we use 60 gradient
+directions for two non-zero-b-values (1000 and 2000 $s/mm^{2}$) and two
```

Comment:
"non-zero-b-values" => "non-zero b-values" (remove the dash between the two parts).